kroxylicious-api/src/main/templates/Kproxy/Filter.ftl

```diff
@@ -68,6 +72,8 @@ public interface ${filterClass} extends Filter {
      * @see io.kroxylicious.proxy.filter Creating Filter Result objects
      * @see io.kroxylicious.proxy.filter  Thread Safety
      */
+     @RunsOnThread("filter thread")
+     @CompletesOnThread("*")
```

Comment:
Do we really want to make this the contract? Should we aim for a stronger contract where this is `@CompletesOnThread("filter thread")` ?

Reply:
My thinking: the runtime should try to protect itself from sloppily written filters. So it would probably need some mitigation in case a filter completes a returned CS on the wrong thread. We means we end up with the code in the runtime in any case, so we might as well let filter authors complete on any thread.